docs/data-sources/dns_acls.md

```diff
@@ -60,6 +60,7 @@ Optional:
 Read-Only:
 
 - `id` (String) The resource identifier.
+- `tags_all` (Map of String) Tagging specifics includes the default tags.
```

Comment:
The description here (and some other resources) is inconsistent with the majority.

Reply:
I'll address this and add tests in a follow up PR